app/Exceptions/BadUrlException.php

```diff
@@ -1,6 +1,9 @@
 <?php
-class FreshRSS_BadUrl_Exception extends FreshRSS_Feed_Exception {
+
+class FreshRSS_BadUrl_Exception extends \Exception {
```

Comment:
Mmm, did you check there is no problem by changing inheritance here? I'm pretty there are some `FreshRSS_Feed_Exception` catch which are used to catch `FreshRSS_BadUrl_Exception` too.